app/data/census_individual.json

```diff
@@ -863,60 +647,127 @@
                                                 }
                                             ],
                                             "q_code": "20",
-                                            "type": "Radio",
-                                            "validation": {
-                                                "messages": {}
-                                            }
+                                            "type": "Radio"
                                         }
-                                    ]
+                                    ],
+                                    "skip_condition": {
+                                        "when": {
+                                            "meta": "region_code",
+                                            "condition": "not equals",
+                                            "value": "GB-WLS"
+                                        }
+                                    }
                                 }
                             ]
                         }
+                    ],
+                    "routing_rules": [
+                        {
+                            "goto": {
+                                "id": "carer",
+                                "when": {
+                                    "id": "country-of-birth-england-answer",
+                                    "condition": "equals",
+                                    "value": "England"
+                                }
+                            }
+                        },
+                        {
+                            "goto": {
+                                "id": "carer",
+                                "when": {
+                                    "id": "country-of-birth-england-answer",
+                                    "condition": "equals",
+                                    "value": "Wales"
+                                }
+                            }
+                        },
+                        {
+                            "goto": {
+                                "id": "carer",
+                                "when": {
+                                    "id": "country-of-birth-england-answer",
+                                    "condition": "equals",
+                                    "value": "Scotland"
+                                }
+                            }
+                        },
+                        {
+                            "goto": {
+                                "id": "carer",
+                                "when": {
+                                    "id": "country-of-birth-england-answer",
+                                    "condition": "equals",
+                                    "value": "Northern Ireland"
+                                }
+                            }
+                        },
+                        {
+                            "goto": {
```

Comment:
Think this was on a different line before. It was on one about england wales scotland or northern ireland but it had them twice